firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/internal/util/conversions.kt

```diff
@@ -135,7 +135,7 @@ internal fun com.google.ai.client.generativeai.common.server.Candidate.toPublic(
 }
 
 internal fun com.google.ai.client.generativeai.common.UsageMetadata.toPublic(): UsageMetadata =
-  UsageMetadata(promptTokenCount, candidatesTokenCount, totalTokenCount)
+  UsageMetadata(promptTokenCount, candidatesTokenCount ?: 0, totalTokenCount)
```

Comment:
Also, I've made this field optional

https://github.com/firebase/firebase-android-sdk/pull/5830/files

`0` would be misleading IMO